packages/block-library/src/post-featured-image/index.php

```diff
@@ -31,7 +31,11 @@ function render_block_core_post_featured_image( $attributes, $content, $block )
 	$overlay_markup = get_block_core_post_featured_image_overlay_element_markup( $attributes );
 
 	if ( $is_link ) {
-		$attr['alt'] = trim( strip_tags( get_the_title( $post_ID ) ) );
+		if( get_the_title( $post_ID ) ) {
+			$attr['alt'] = trim( strip_tags( get_the_title( $post_ID ) ) );
+		} else {
+			$attr['alt'] = esc_attr__( 'Untitled post ' ) . $post_ID;
```

Comment:
I do not think this escape is needed here as we are not outputting it at this point. I would hope these attributes get filtered before output later.

```suggestion
			$attr['alt'] = wp_sprintf( __( 'Untitled post %d' ), $post_ID );
```

https://developer.wordpress.org/reference/functions/wp_sprintf/
https://developer.wordpress.org/reference/functions/__/

I would give this a test but this looks good from docs.

Reply:
Good catch. 
I have used sprintf instead of wp_sprintf.
I did not know wp_sprintf existed :)